documentation/modules/security/proc-accessing-kafka-using-routes.adoc

```diff
@@ -45,36 +66,90 @@ spec:
-For example, `my-cluster-kafka-listener1-bootstrap-myproject` (_CLUSTER-NAME_-kafka-_LISTENER-NAME_-bootstrap-_NAMESPACE_). Be careful that the whole length of the address does not exceed a maximum limit of 63 characters.
 
 . Create or update the resource.
 +
 [source,shell,subs=+quotes]
+----
 kubectl apply -f _<kafka_configuration_file>_
+----
++
+A cluster CA certificate to verify the identity of the kafka brokers is created in the secret `my-cluster-cluster-ca-cert`.
 +
-`ClusterIP` type services are created for each Kafka broker, as well as an external _bootstrap service_.
-The services route the traffic from the OpenShift Routes to the Kafka brokers.
-An OpenShift `Route` resource is also created for each service to expose them using the HAProxy load balancer.
-DNS addresses used for connection are propagated to the `status` of each service.
+`ClusterIP` type services are created for each Kafka broker, as well as an external bootstrap service.
 +
-The cluster CA certificate to verify the identity of the kafka brokers is also created in the secret `_<cluster_name>_-cluster-ca-cert`.
+A `route` is also created for each service, with a DNS address (host/port) to expose them using the default OpenShift HAProxy router.
++
+The routes are preconfigured with TLS passthrough. 
++
+.Routes created for the bootstraps and brokers
+[source,shell]
+----
+NAME                                  HOST/PORT                                                   SERVICES                              PORT  TERMINATION
+my-cluster-kafka-listener1-0          my-cluster-kafka-listener1-0-my-project.router.com          my-cluster-kafka-listener1-0          9094  passthrough
+my-cluster-kafka-listener1-1          my-cluster-kafka-listener1-1-my-project.router.com          my-cluster-kafka-listener1-1          9094  passthrough
+my-cluster-kafka-listener1-2          my-cluster-kafka-listener1-2-my-project.router.com          my-cluster-kafka-listener1-2          9094  passthrough
+my-cluster-kafka-listener1-bootstrap  my-cluster-kafka-listener1-bootstrap-my-project.router.com  my-cluster-kafka-listener1-bootstrap  9094  passthrough
+----
++
+The DNS addresses used for client connection are propagated to the `status` of each route.
++
+.Example status for the bootstrap route
+[source,yaml]
+----
+status:
+  ingress:
+    - host: >-
+        my-cluster-kafka-listener1-bootstrap-my-project.router.com
+ # ...
+----
 
-. Retrieve the address of the bootstrap service you can use to access the Kafka cluster from the status of the `Kafka` resource.
+. Use a target broker to check the client-server TLS connection on port 443 using the OpenSSL `s_client`.  
 +
-[source,shell,subs=+quotes]
-kubectl get kafka _<kafka_cluster_name>_ -o=jsonpath='{.status.listeners[?(@.name=="_<listener_name>_")].bootstrapServers}{"\n"}'
+[source,shell]
+----
+openssl s_client -connect my-cluster-kafka-0:443 -servername my-cluster-kafka-0 -showcerts
```

Comment:
Again, I think you should use the Route address here: `my-cluster-kafka-listener1-0-my-project.router.com`